lib/executables/process.js

```diff
@@ -1,19 +1,60 @@
+const _ = require('lodash');
+const fs = require('fs');
+const vm = require('vm');
+const request = require('co-request');
+const { messages } = require('elasticio-node');
 const prompts = require('../helpers/prompts.js');
 const utils = require('../helpers/utils.js');
 const { Emitter, EmitterError } = require('../helpers/emitter.js');
 
+function wait(timeout) {
+  return new Promise((ok) => {
+    setTimeout(() => {
+      this.logger.debug('Done wait');
+      ok();
+    }, timeout);
+    this.logger.debug('Start wait sec=%s', timeout);
+  });
+}
+
 exports.runProcess = async function runProcess(componentPath, fixtureKey, action) {
   const actionData = await prompts.getActionPath(componentPath, action);
   const { path } = actionData;
   const actionName = actionData.name;
 
   const fixture = await prompts.setupFixture(fixtureKey, componentPath);
   utils.validateFixture(componentPath, fixture, actionName);
-  const component = utils.resolveComponent(path);
+  const vmExports = {};
+  const context = vm.createContext({
+    // Node globals
+    Buffer,
+    clearInterval,
+    clearTimeout,
+    console,
+    exports: vmExports,
+    global: {},
+    module: { exports: vmExports },
+    require,
+    setInterval,
+    setTimeout,
+    URL,
+    URLSearchParams,
+
+    // elasticio specific functionality
```

Comment:
In the case of components we also expect this to be imported by the client code.